### ISSUE TITLE
Creating PR from github desktop causing issue V 1.1.0

### ISSUE THREAD

**Issue by Pritraj** [desktop/desktop#4141]
CTRL + R OR 'Show pull request' sends me to already merged/closed request.

Facing from V 1.1.0

It should send me to new/previously created pull request AND NOT to already processed request.

Also, it doesn't update PR in desktop
![image](https://user-images.githubusercontent.com/18588781/36829029-32d80e9c-1d43-11e8-86b3-cdfdf611d27f.png)

342 is already merged in my case and hence it sends me to 342 always on creating PR.

**Comment by iAmWillShepherd** [comment]
Thanks for taking the time to open an issue! 🙇 

If you would like to create a new PR, you must create a new branch. The workflow we provide at the moment assumes that once your PR is merged, you will create a new branch and create a PR from there.
